tests/RequestsTest.php

```diff
@@ -169,4 +171,30 @@ public function testTimeoutException() {
 		$this->expectExceptionMessage('timed out');
 		Requests::get(httpbin('/delay/3'), array(), $options);
 	}
+
+	/**
+	 * @covers \WpOrg\Requests\Requests::has_capabilities
+	 */
+	public function testHasCapabilitiesSucceedsForDetectingSsl() {
+		if (!extension_loaded('curl') && !extension_loaded('openssl')) {
+			$this->markTestSkipped('Testing for SSL requires either the curl or the openssl extension');
+		}
+		$this->assertTrue(Requests::has_capabilities(array(Capability::SSL => true)));
+	}
+
+	/**
+	 * @covers \WpOrg\Requests\Requests::has_capabilities
+	 */
+	public function testHasCapabilitiesFailsForUnsupportedCapabilities() {
+		$transports = new ReflectionProperty(Requests::class, 'transports');
+		$transports->setAccessible(true);
+		$transports->setValue(array(TestTransportMock::class));
+
+		$result = Requests::has_capabilities(array('time-travel' => true));
+
+		$transports->setValue(null);
```

Comment:
```suggestion
		$transports->setValue(array());
```

The default value in the `Requests` class is an empty array, so let's revert back to that instead of introducing a new variable type (`null`) which may cause side effects.

https://github.com/WordPress/Requests/blob/afa69a60b32c5b9263266711e5f635938c2fc39f/src/Requests.php#L175